### DISCUSSION TITLE
Print timestamp for when a command completes

### DISCUSSION THREAD

**Discussion by cmawhorter** [starship/starship#4899] [Q&A] [answered]
I've looked through the docs and tried a few things but I've never really done any starship config before, and I'm coming up empty.

I want to print a timestamp after a command runs.

It seems natural to pair this with command duration and with that on, it shows something like "took 2s" or whatever. How do I get it to say "took 2s at 14:53" ?

**Comment by chipbuster** [comment, marked as answer]
I do this with the time module, but that triggers on every draw.

I'm not aware of a way to do this while only printing a time when the cmd_duration module triggers (perhaps there's a way to do it with a custom module, but nothing built-in).

**Reply by cmawhorter** [reply]
i don't know why, but for some reason i thought the time module behaved like a live clock for some reason. but that combined with cmd_duration is good enough for me
